ElectronClient/app.js

```diff
@@ -302,6 +313,17 @@ class Application extends BaseApplication {
 			this.updateMenuItemStates(newState);
 		}
 
+		if (['THEME_TOGGLE'].indexOf(action.type) >= 0) {
+			Setting.setValue('isNightMode', newState.isNightMode);
+			const activeDarkTheme = Setting.value('preferredDarkTheme');
+			const activeLightTheme = Setting.value('preferredLightTheme');
+			newState.isNightMode ? Setting.setValue('theme', activeDarkTheme) : Setting.setValue('theme', activeLightTheme);
```

Comment:
This should be put in an if else construct, so you don't need to retrieve and store unnecessary setting.  Also should you set the theme before you toggle the night mode?  ie should the order of your setting changes be reversed?

Reply:
yes you are right using if else statement save the memory, I used ternary operator keeping "SHORT CODE" in mind.I will improve this in new commit. 

>should the order of your setting changes be reversed?
No, it works like if `isNightMode` is true then theme will be set to `preferredDarkTheme` and for false value theme will be set to `preferredLightTheme`